client/src/app/components/stardust/TransactionPayloadProps.tsx

```diff
@@ -32,5 +32,10 @@ export interface TransactionPayloadProps {
      * The header title of this section.
      */
     header: string;
+
+    /**
+     * Is block conflicting.
+     */
+    isConflictingBlock?: boolean;
```

Comment:
Let's call this `isLinksDisabled`. Payload doesn't have to have notion of Block.

Reply:
Done.